clock/src/main/kotlin/io/spine/examples/pingh/clock/ClockContext.kt

```diff
@@ -46,7 +47,8 @@ private val actor = UserId.newBuilder()
 /**
  * Emits the `TimePassed` event that contains the current time.
  */
-internal fun emitTimePassedEvent() {
+@TestOnly
```

Comment:
Why is it so? If it is test-only, it should be in the respective `test` sources.

I think you wanted it to be `@VisibleForTesting`. Which would mean "normally, the visibility of this item should be lower (e.g. `internal`), but for the sake of testing it is higher (e.g. `public`).

Also, let's not use JetBrains annotations.